docs/StardustDocs/cfg/resize-iframes.js

```diff
@@ -46,59 +46,64 @@
         }
 
         function doObserveIFrame(el) {
-            resize_iframe_out(el)
-            let observer = new MutationObserver(function (mutations) {
-                var resize = false
+            resize_iframe_out(el);
+            const mutationObserver = new MutationObserver((mutations) => {
+                let resized = false;
                 for (let mutation of mutations) {
                     // skip attribute changes except open to avoid loop
                     // (callback sees change in iframe, triggers resize, sees change...)
                     if (mutation.type === 'attributes') {
                         if (mutation.attributeName === 'open') {
-                            resize_iframe_out(el)
-                            resized = true
+                            resize_iframe_out(el);
+                            resized = true;
```

Comment:
on a second thought now `resized = true` triggers second call of `resize_iframe_out` which doesn't make much sense to me

Reply:
so should we remove them altogether?

Reply:
agreed, I'll also rename it to `resize` then, since it defines whether the resize will happen or not.